packages/showcase/src/app/components/accordion/metadata.ts

```diff
@@ -0,0 +1,343 @@
+import { ComponentMetadata } from "../../../common/types";
+import { chameleonImportType } from "../../../services/template-language/create-template";
+
+import { accordionCommonChildren, accordionHeaderSlotChildren } from "./common";
+import {
+  accordionModel,
+  accordionWithIconsModel,
+  accordionHeaderSlotModel,
+  accordionWithIconsModelForMetadata
+} from "./models";
+
+const CH_TAG = "ch-accordion-render";
+const FILLED_CLASS = "accordion-filled";
+const OUTLINED_CLASS = "accordion-outlined";
+const ELEVATION_1_CLASS = "elevation-1";
+const ELEVATION_2_CLASS = "elevation-2";
+const ELEVATION_3_CLASS = "elevation-3";
+const WITH_GAP_CLASS = "with-gap";
+
+export const accordionMetadata = {
+  title: "Accordion",
+  description:
+    "An accordion is a UI component that organizes content into collapsible sections, allowing users to expand or collapse panels as needed. It helps keep interfaces clean by displaying only relevant information while providing quick access to additional details.",
+
+  codeSnippets: {
+    // Filled Cases
+
+    filled: {
+      linkId: "accordion-filled",
+      title: "1.1. Filled",
+      states: [
+        {
+          name: "genexusObjects",
+          type: "AccordionModel",
+          value: accordionModel
+        }
+      ],
+      imports: [chameleonImportType("AccordionModel")],
+      template: {
+        tag: CH_TAG,
+        class: `${FILLED_CLASS}`,
+        properties: [{ name: "model", value: "genexusObjects", state: true }],
+        children: accordionCommonChildren
+      }
+    },
+
+    filledDisabled: {
+      linkId: "accordion-filled-disabled",
+      title: "1.2. Filled: Disabled",
+      states: [
+        {
+          name: "genexusObjects",
+          type: "AccordionModel",
+          value: accordionModel
+        }
+      ],
+      imports: [chameleonImportType("AccordionModel")],
+      template: {
+        tag: CH_TAG,
+        class: `${FILLED_CLASS}`,
+        properties: [
+          { name: "model", value: "genexusObjects", state: true },
+          { name: "disabled", value: true }
+        ],
+        children: accordionCommonChildren
+      }
+    },
+
+    filledWithIcons: {
+      linkId: "accordion-filled-with-icons",
+      title: "1.3. Filled With Icons",
+      states: [
+        {
+          name: "genexusObjects",
+          type: "AccordionModel",
+          value: accordionWithIconsModelForMetadata
+        }
+      ],
+      imports: [
+        chameleonImportType("AccordionModel"),
+        'import { getIconPath, getIconPathExpanded } from "@genexus/mercury/assets-manager.js";'
```

Comment:
The `getIconPathExpanded` method is not supported in the accordion.

```suggestion
        'import { getIconPath } from "@genexus/mercury/assets-manager.js";'
```